### PR TITLE
reformat code

### LINKED ISSUE #105: Refine ClangFormat
https://zed0.co.uk/clang-format-configurator/

### PR DESCRIPTION
**Pull request by zpl-zak**

### CODE REVIEW

File: code/framework/src/networking/network_client.cpp [hunk 1]
```diff
@@ -101,55 +101,55 @@ namespace Framework::Networking {
...
 
         case ID_CONNECTION_BANNED: {
             if (_onPlayerDisconnectedCallback) {
                 _onPlayerDisconnectedCallback(_packet, Messages::BANNED);
             }
             _state = DISCONNECTED;
             return true;
-        };
+        }
         }
```
Comment (Deewarz) [inline code/framework/src/networking/network_client.cpp]: two brackets at the same ident?

File: code/framework/src/utils/time.cpp [hunk 2]
```diff
@@ -12,17 +12,16 @@ namespace Framework::Utils {
...
         return (GetMilliseconds(a) - GetMilliseconds(b));
     }
-    int64_t Time::GetMilliseconds(Time::TimePoint a) {
+    int64_t Time::GetMilliseconds(TimePoint a) {
         return std::chrono::duration_cast<std::chrono::milliseconds>(a.time_since_epoch()).count();
     }
-    int32_t Time::Compare(Time::TimePoint a, Time::TimePoint b) {
+    int32_t Time::Compare(TimePoint a, TimePoint b) {
         if (a == b)
             return 0;
```
Comment (Deewarz) [inline code/framework/src/utils/time.cpp]: always enforce brackets?

File: code/framework/src/world/modules/modules_impl.cpp [hunk 3]
```diff
@@ -173,19 +173,17 @@ namespace Framework::World::Modules {
...
             if (!msg->Valid()) {
                 return;
             }
 
             const auto e = worldEngine->GetEntityByServerID(msg->GetServerID());
 
-            if (!e.is_alive()) {
-                return;
-            }
+            if (!e.is_alive()) {}
```
Comment (Deewarz) [inline code/framework/src/world/modules/modules_impl.cpp]: could be removed then

File: code/framework/src/networking/network_client.cpp (hunk 1 above)
Comment (zpl-zak) [reply code/framework/src/networking/network_client.cpp]: yes, notice switch case is pushed down, this is fine.

**Comment by zpl-zak** [comment]
Closes #105 hehe

**Pull request merged**